packages/server/lib/browsers/webkit.ts

```diff
@@ -30,32 +31,7 @@ export async function open (browser: Browser, url, options: any = {}, automation
     headless: browser.isHeadless,
   })
 
-  let pwPage: playwright.Page
-
-  async function resetPage (_url) {
```

Comment:
@lmiller1990 here is the `reset` refactor requested: https://github.com/cypress-io/cypress/pull/15533#discussion_r943050562